vscode/interop/interop.mli

```diff
@@ -20,12 +20,10 @@ module Regexp : sig
   val t_of_js : Ojs.t -> Js_of_ocaml.Regexp.regexp
 end
 
-module Dict : sig
-  include Map.S with type key = string
+module JsDict : sig
+  type 'a t = (string, 'a) Core_kernel.Map.Poly.t
 
-  val t_to_js : ('a -> Ojs.t) -> 'a t -> Ojs.t
+  val t_to_js : ('a -> Ojs.t) -> (string, 'a) Core_kernel.Map.Poly.t -> Ojs.t
 
-  val t_of_js : (Ojs.t -> 'a) -> Ojs.t -> 'a t
-
-  val of_alist : (string * 'a) list -> 'a t
+  val t_of_js : (Ojs.t -> 'a) -> Ojs.t -> (string, 'a) Core_kernel.Map.Poly.t
 end
```

Comment:
I think you have some leftover changes here.

Reply:
Got rid of them.